### PR TITLE
UI Rework

### PR DESCRIPTION
**Pull request by Str4ky**
Did a rework of the UI:

- New placements of elements
- Mario font added
- Window not resizable (coz the app will not like that)

### CODE REVIEW

File: Assets/logo.png
Comment (SandeMC) [inline Assets/logo.png:1]: wouldn't it be better if the "proximity chat client v1.0" was an in-scene text for easier editing?

**Comment by PabloMK7** [comment]
Keep in mind that the UI will change on the next release

File: Assets/logo.png
Comment (Str4ky) [reply Assets/logo.png:1]: I don't know

**Comment by Str4ky** [comment]
I saw the change, I will try to update it but I will have to learn 'bout saving things in Unity between scenes and also fixes the bugs I had if I encounter them again. Unless you want to rewrite the UI yourself, that way I can like delete my fork and see what you can make

**Comment by PabloMK7** [comment]
I don't have plans to rework the UI. You can't use multiple scenes as that will destroy everything, including the logic handing voice chat.

**Comment by Ksplover2343** [comment]
is it possible for future releases to have MacOS support? just a quick question, as i wanna communicate with people (and hear them raging)

**Comment by Str4ky** [comment]
> I don't have plans to rework the UI. You can't use multiple scenes as that will destroy everything, including the logic handing voice chat.

I see, I'm gonna do some tests and see if I can come up with something nice

**Pull request closed without merging**